data/hai/unfettered jobs.txt

```diff
@@ -312,7 +312,7 @@ mission "Unfettered Small Medicals Rush"
 	job
 	repeat
 	clearance "After speaking with the Unfettered and explaining your presence, they give you permission to land with the much needed medical supplies."
-	name "Express medical supplies to <planet>"
+	name "Express medical supplies from <planet stopovers>"
```

Comment:
```suggestion
	name "Express medical stock from <planet stopovers>"
```
@Saugia mentioned that planet names get clipped with this one. This one suggested by doomkorath is a little shorter; thoughts?

Reply:
![UH1](https://github.com/endless-sky/endless-sky/assets/93169396/1dbfef8d-2541-4389-9643-b89c631e5df3)
![UH2](https://github.com/endless-sky/endless-sky/assets/93169396/6545fb3f-365f-4c49-8284-a325a197fc67)

That appears to fix the overlap. The problematic names fit now, and so do the others it seems.